Gemfile

```diff
@@ -15,3 +15,4 @@ gem "colorize"
 gem "capybara"
 gem "poltergeist"
```

Comment:
Presumably these two can be removed now?

(NB: the inconsistent formatting here implies that this Gemfile also needs brought up to current standards, perhaps as a separate PR?)